src/routes/console/project-[project]/storage/bucket-[bucket]/create-file/step1.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+    import { CustomId } from '$lib/components';
+    import { Pill } from '$lib/elements';
+    import { FormList, InputFile } from '$lib/elements/forms';
+    import WizardStep from '$lib/layout/wizardStep.svelte';
+    import { bucket } from '../store';
+    import { createFile } from './store';
+
+    let showCustomId = false;
+</script>
+
+<WizardStep>
+    <svelte:fragment slot="title">Upload a File</svelte:fragment>
+    <svelte:fragment slot="subtitle">Upload a file to add it to your bucket.</svelte:fragment>
+    <FormList>
+        <div>
+            <InputFile
+                required
+                bind:files={$createFile.files}
+                allowedFileExtensions={$bucket.allowedFileExtensions}
+                maxSize={$bucket.maximumFileSize} />
+        </div>
+
+        {#if !showCustomId}
+            <div>
+                <Pill button on:click={() => (showCustomId = !showCustomId)}>
+                    <span class="icon-pencil" aria-hidden="true" /><span class="text">
+                        File ID
+                    </span>
+                </Pill>
+            </div>
+        {:else}
+            <div class="custom-id-wrapper">
+                <CustomId bind:show={showCustomId} name="File" bind:id={$createFile.id} />
+            </div>
+        {/if}
+    </FormList>
+</WizardStep>
+
+<style lang="scss">
+    .custom-id-wrapper :global(.is-inner-modal) {
+        inline-size: auto;
+    }
```

Comment:
@TGlide Is this following the design? Because we have "normal" sized innerModals in some other wizards :thinking: 
![image](https://user-images.githubusercontent.com/45005369/225695426-b1f8dace-4b23-480e-b3a4-6779f5c737cd.png)

@heyCarla do the other customIDs in wizards need to be full width for consistency? In that case, we can add a full-width parameter to the component :)

Reply:
> @TGlide Is this following the design? Because we have "normal" sized innerModals in some other wizards 🤔 ![image]
> 
> @heyCarla do the other customIDs in wizards need to be full width for consistency? In that case, we can add a full-width parameter to the component :)

Yep, these custom ID components should be full-width in the wizards. (P.S. where do we show the "normal" sized ones? I thought they should always be full-width to whatever container they're nested in, but maybe there's a use case I've forgotten 🙃 )